tests/index.test.ts

```diff
@@ -1,48 +1,46 @@
-import { existsSync } from 'fs';
-import { resolve } from 'path';
+import { existsSync, readdirSync } from 'node:fs';
+import { resolve } from 'node:path';
+
+import { it, expect } from 'vitest';
 
 import plugin from '../lib';
 
-const numberOfRules = 28;
 const ruleNames = Object.keys(plugin.rules);
 
-// eslint-disable-next-line jest/expect-expect
+it('should export all existing rules', () => {
```

Comment:
This new test will take care of making sure that all rules within `libs/rules` are actually re-exported, so can be consumed by users installing the plugin.

The purpose for this check is to improve the DX when working on new rules, since it's quite easy to forget re-exporting the rules to the public API of the plugin.

Here, a demo of the test catching a rule not re-exported properly:

https://github.com/user-attachments/assets/f92eb389-e323-4d2b-aa72-72e351ece88d